app.py

```diff
@@ -24,7 +24,16 @@ def create_movies_form():
 def create_movie():
     # TODO: Feature 2
     # After creating the movie in the database, we redirect to the list all movies page
-    return redirect('/movies')
+
+    title = request.form.get('title', None)
+    director = request.form.get('director', None)
+    rating = request.form.get('rating', None)
+
+    if (title == None or director==None or int(rating)<1 or int(rating)>5):
```

Comment:
You might want to also check if `title` and `director` are equal to an empty string